bundles/org.openhab.binding.resol/pom.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>2.5.6-SNAPSHOT</version>
```

Comment:
```suggestion
    <version>2.5.8-SNAPSHOT</version>
```

Reply:
This killed me ;-( Again. As simply changing the version in the pom did not work due to the version conflict I tried to update to the latest version in the 2.5.x branch and now my environment is broken. On the command line my binding compiles properly but ecplise is full of errors like "Failed to create implicit repository for m2e project org.openhab.demo.app (implicit)"

I am about to give up...